index.php

```diff
@@ -0,0 +1,124 @@
+            Calathea medallion
+            <h3>NICKNAME:</h3>
+            Prayer plant
+            <h3>PLANT TYPE:</h3>
+            Perennial, indoor
+            <h3>PLANT HEIGHT:</h3>
+            76.3 cm
+            <h3>POISENOUS:</h3>
+            No
+        </div>
+
+        <div class="plants">
+            <img class="elephant_ear" src="images/elephant-ear-plant.jpg" alt="beautiful elephant ear plant with a blue background">
+            <h2>Elephant Ear Plant</h2>
+            <h3>BOTANICAL NAME:</h3>
+            Alocasia zebrina
+            <h3>NICKNAME:</h3>
+            Elephant ear plant
+            <h3>PLANT TYPE:</h3>
+            Perennial, indoor
+            <h3>PLANT HEIGHT:</h3>
+            55.5 cm
+            <h3>POISENOUS:</h3>
+            Yes
+        </div>
+
+        <div class="plants">
+            <img class="spider_plant" src="images/curly-spider-plant.jpg" alt="curly spider plant in a grey pot">
+            <h2>Curly Spider Plant</h2>
+            <h3>BOTANICAL NAME:</h3>
+            Chlorophytum comosum
+            <h3>NICKNAME:</h3>
+            Curly spider plant; Variegated spider plant; Airplane plant
+            <h3>PLANT TYPE:</h3>
+            Perennial indoor
+            <h3>PLANT HEIGHT:</h3>
+            30 cm
+            <h3>POISENOUS:</h3>
+            No
+        </div>
+
+        <div class="plants">
+            <img class="ivy" src="images/ivy.jpg" alt="english ivy in a grey pot with white background">
+            <h2>English Ivy Plant</h2>
+            <h3>BOTANICAL NAME:</h3>
+            Hedera helix
+            <h3>NICKNAME:</h3>
+            English ivy; Common ivy; European ivy
+            <h3>PLANT TYPE:</h3>
+            Evergreen climber
+            <h3>PLANT HEIGHT:</h3>
+            34.5
+            <h3>POISENOUS:</h3>
+            Yes
+        </div>
+
+        <div class="plants">
+            <img class="snake_plant" src="images/snake-plant.jpg" alt="snake plant also called mother in laws tongues in a dark grey pot">
+            <h2>Snake Plant</h2>
+            <h3>BOTANICAL NAME:</h3>
+            Sansevieria trifasciata
+            <h3>NICKNAME:</h3>
+            Snake plant; Mother-in-Laws tongue; Vipers bowst
+            <h3>PLANT TYPE:</h3>
+            Perennial, indoor
+            <h3>PLANT HEIGHT:</h3>
+            64.3 cm
+            <h3>POISENOUS:</h3>
+            Yes
+        </div>
+
+        <div class="plants">
+            <img class="cheese_plant" src="images/cheeseplant.jpg" alt="cheese plant in a traditional plastic pot">
+            <h2>Cheese Plant</h2>
+            <h3>BOTANICAL NAME:</h3>
+            Monstera deliciosa
+            <h3>NICKNAME:</h3>
+            Cheese plant; Ceriman; Custard plant; Indian ivy; Fruit salad plant; Mexican breadfruit
+            <h3>PLANT TYPE:</h3>
+            Evergreen shrub, indoor
+            <h3>PLANT HEIGHT:</h3>
+            76.3cm
+            <h3>POISENOUS:</h3>
+            yes
+
```

Comment:
oh sorry, i mean this part. there is an opening div on line 99 and then on this line 113 there is a closing div tag separate from the code above after a blank line of code on line 112